src/main/resources/geoserver/layer_creation.xml

```diff
@@ -1,51 +1,187 @@
 <featureType>
 	<name>LAYER_NAME</name>
-	<enabled>true</enabled>
-	<title>Beachfront Detections</title>
+	<nativeName>LAYER_NAME</nativeName>
+	<namespace>
+		<name>piazza</name>
```

Comment:
Wouldn't we want to reference the configured workspace name here?

Reply:
Yes, yes we would.
Should be fixed now.